distribution/ddf-common/src/main/resources/security/default.policy

```diff
@@ -359,6 +359,23 @@ grant {
     permission java.io.FilePermission "/jenkins/-", "read";
 }
 
+// Permissions required by the wrapper:install command
+grant codeBase "file:/org.apache.karaf.wrapper.core/org.apache.karaf.shell.core/org.apache.karaf.shell.ssh/org.apache.sshd.core" {
+    permission java.io.FilePermission "<<ALL FILES>>", "execute";
+}
+grant codeBase "file:/org.apache.karaf.wrapper.core/org.apache.karaf.shell.ssh/org.apache.sshd.core" {
+    permission java.io.FilePermission "${ddf.home.perm}bin${/}*”, "read, write";
+    permission java.io.FilePermission "${ddf.home.perm}etc${/}*”, "read, write";
```

Comment:
Do these really need read/write to everything in etc and bin?

Reply:
The wrapper:install command creates a set of files in etc and bin which are named based on a parameter we pass in. I gave it read/write within the directories because we can't predict the name.